### ISSUE TITLE
Deprecate distribution?

### ISSUE THREAD

**Issue by Grinnz** [pplu/mojo-ioloop-future#2]
Since Future::Mojo serves the original purpose of this dist, do you think it could be marked deprecated as described here: https://neilb.org/2015/01/17/deprecated-metadata.html and perhaps suggest Future::Mojo instead?

**Comment by pplu** [comment]
Yes. If you send me a PR, I'll gladly apply and upload to CPAN. Feel free to change the POD to point to the better Future::Mojo
